content/doc/book/system-administration/reverse-proxy-configuration-with-jenkins/reverse-proxy-configuration-iptables.adoc

```diff
@@ -207,3 +207,28 @@ do SSL or not)
 firewalld will then create the required iptables rules so that incoming
 connections on port 80 are forwarded to jenkins on 8080 (and 443 is
 forwarded to 8443).
+
+
+== Unix Domain Sockets support
+The addition of Unix domain sockets support for Jenkins when configured behind an Iptables reverse proxy. 
+This feature allows for enhanced communication between Jenkins and the reverse proxy, 
+improving performance and security.
+
+=== What does Unix Domain Sockets Support Mean for Iptables?
+With Unix domain sockets support, Iptables can now directly communicate with Jenkins via Unix domain sockets, 
+providing a more efficient and secure connection compared to traditional TCP sockets. 
+This eliminates the need for network-level communication between Jenkins and the reverse proxy, reducing overhead and potential vulnerabilities.
+
+=== Example Configuration:
+To configure Unix domain sockets with Iptables for Jenkins, you can use the following example code:
+
+[source]
+----
+# Redirect traffic from port 80 to the Unix domain socket
+iptables -t nat -A PREROUTING -i eth0 -p tcp --dport 80 -j REDIRECT --to-ports <socket_path>
+# Redirect traffic from port 443 to the Unix domain socket
+iptables -t nat -A PREROUTING -i eth0 -p tcp --dport 443 -j REDIRECT --to-ports <socket_path>
+----
+
+Replace <socket_path> with the path to your Unix domain socket. 
+This will redirect incoming traffic on ports 80 and 443 to the Unix domain socket, allowing Jenkins to receive requests directly.
```

Comment:
This does not appear to align with the [Iptables documentation regarding Unix domain sockets](https://linux.die.net/man/7/unix). It does appear that redirecting the traffic to a port would follow those instructions, but this process looks to be more involved than just that.

Would you be able to share the source for this configuration to compare?

Reply:
For this part even I didn't had knowledge so I had taken it from an answer, I was not able to find that again but will post if I get some info to it more

Reply:
Hi @pyther-hub, did you happen to find the original source that had this information? I'd be very interested in reviewing it myself so that I can better understand.